### PR TITLE
PS-9268 Document SEQUENCE_TABLE() -> PERCONA_SEQUENCE_TABLE() transi…

### PR DESCRIPTION
**Pull request by patrickbirch**
…tion

 On branch ps-9268
	modified:   docs/sequence-table.md

### CODE REVIEW

File: docs/percona-sequence-table.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+| Generates Sequences       | Acts as an inline table-valued function that generates a sequence of numbers.                     |
+| Table-Valued Function     | Unlike traditional scalar functions, `PERCONA_SEQUENCE_TABLE()` returns a virtual table with a single column named `value` containing the generated sequence. |
+| Simpler Syntax            | Simplifies queries that need to generate predictable sequences of numbers.                        |
+| Flexibility               | Allows dynamic definition of sequences within queries, offering more control compared to pre-defined tables for sequences. |
+| Predefined Sequence       | Does not manage sequences like Oracle or PostgreSQL; instead, it allows definition and generation of sequences within a `SELECT` statement. |
+| Customization             | Enables customization of starting value, increment/decrement amount, and number of values to generate. |
+
+## Version update
+
+Percona Server for MySQL 8.0.37 deprecated `SEQUENCE_TABLE()`, and Percona may remove this variable in a future release. We recommend that you use `PERCONA_SEQUENCE_TABLE()` instead.
```
Comment (percona-ysorokin) [inline docs/percona-sequence-table.md]:
```suggestion
Percona Server for MySQL 8.0.37 deprecated `SEQUENCE_TABLE()`, and Percona may remove this function in a future release. We recommend that you use `PERCONA_SEQUENCE_TABLE()` instead.
```

File: docs/percona-sequence-table.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+## Syntax
+
+As with any [derived tables](https://dev.mysql.com/doc/refman/8.0/en/derived-tables.html), a table function requires an [alias](https://dev.mysql.com/doc/refman/8.0/en/identifiers.html) in the `SELECT` statement.
+
+The result set is a single column with the predefined column name `value` of type `BIGINT UNSIGNED`. You can reference the `value` column in `SELECT` statements. The following statements are valid. Using `n` as the number of generated values, the following is the basic syntax:
+
+* SEQUENCE_TABLE(n) [AS] alias
```
Comment (percona-ysorokin) [inline docs/percona-sequence-table.md]:
```suggestion
* PERCONA_ SEQUENCE_TABLE(n) [AS] alias
```

File: docs/percona-sequence-table.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+
+```{.text .no-copy}
+SELECT … FROM PERCONA_SEQUENCE_TABLE(n) [AS] alias
+
+PERCONA_SEQUENCE_TABLE(n>) [AS] alias
```
Comment (percona-ysorokin) [inline docs/percona-sequence-table.md]:
```suggestion
PERCONA_SEQUENCE_TABLE(n) [AS] alias
```

File: docs/percona-sequence-table.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+SELECT <expr(value)> FROM PERCONA_SEQUENCE_TABLE(n) AS tt;
+```
+
+The first number in the series, the initial term, is defined as `0`, and the series ends with a value less than `n`.
+
+## Example usage
+
+Using `PERCONA_SEQUENCE_TABLE()`:
+
+```{.bash data-prompt="mysql>"}
```
Comment (percona-ysorokin) [inline docs/percona-sequence-table.md]: This example is too weird and syntactically incorrect.

File: docs/sequence-table.md [hunk 5]
```diff
@@ -1,40 +1,70 @@
...
+| Simpler Syntax            | Simplifies queries that need to generate predictable sequences of numbers.                        |
+| Flexibility               | Allows dynamic definition of sequences within queries, offering more control compared to pre-defined tables for sequences. |
+| Predefined Sequence       | Does not manage sequences like Oracle or PostgreSQL; instead, it allows definition and generation of sequences within a `SELECT` statement. |
+| Customization             | Enables customization of starting value, increment/decrement amount, and number of values to generate. |
 
-The [JSON_TABLE()](https://dev.mysql.com/doc/refman/8.0/en/json-table-functions.html) is the only table function available in Oracle MySQL Server. `JSON_TABLE` and `SEQUENCE_TABLE()` are the only table functions available in Percona Server.
+## Version update
 
-The basic syntax is the following:
+Percona Server for MySQL 8.0.37 deprecated `SEQUENCE_TABLE()`, and this variable can be removed in a future release. We recommend that you use [`PERCONA_SEQUENCE_TABLE()`] instead.
```
Comment (percona-ysorokin) [inline docs/sequence-table.md]:
```suggestion
Percona Server for MySQL 8.0.37 deprecated `SEQUENCE_TABLE()`, and this function can be removed in a future release. We recommend that you use [`PERCONA_SEQUENCE_TABLE()`] instead.
```

File: docs/sequence-table.md [hunk 6]
```diff
@@ -1,40 +1,70 @@
...
+| Simpler Syntax            | Simplifies queries that need to generate predictable sequences of numbers.                        |
+| Flexibility               | Allows dynamic definition of sequences within queries, offering more control compared to pre-defined tables for sequences. |
+| Predefined Sequence       | Does not manage sequences like Oracle or PostgreSQL; instead, it allows definition and generation of sequences within a `SELECT` statement. |
+| Customization             | Enables customization of starting value, increment/decrement amount, and number of values to generate. |
 
-The [JSON_TABLE()](https://dev.mysql.com/doc/refman/8.0/en/json-table-functions.html) is the only table function available in Oracle MySQL Server. `JSON_TABLE` and `SEQUENCE_TABLE()` are the only table functions available in Percona Server.
+## Version update
 
-The basic syntax is the following:
+Percona Server for MySQL 8.0.37 deprecated `SEQUENCE_TABLE()`, and this function can be removed in a future release. We recommend that you use [`PERCONA_SEQUENCE_TABLE()`] instead.
```
Comment (alina-derkach-oaza) [inline docs/sequence-table.md]: [`PERCONA_SEQUENCE_TABLE()`] -> add the link

File: docs/sequence-table.md [hunk 7]
```diff
@@ -153,4 +193,7 @@ mysql> SELECT * FROM t1;
     +-------+
     ```
 
-There are many uses for a sequence when populating tables.
+Sequences are useful for various purposes, such as populating tables and generating test data.
+
+[`JSON_TABLE()`]: https://dev.mysql.com/doc/refman/8.0/en/json-table-functions.html
+[`PERCONA_SEQUENCE_TABLE()`]:
```
Comment (alina-derkach-oaza) [inline docs/sequence-table.md]: Add the link to [`PERCONA_SEQUENCE_TABLE()`]:

File: docs/sequence-table.md [hunk 8]
```diff
@@ -1,40 +1,70 @@
...
 
-As with any [derived tables](https://dev.mysql.com/doc/refman/8.0/en/derived-tables.html), a table function requires an [alias](https://dev.mysql.com/doc/refman/8.0/en/identifiers.html) in the `SELECT` statement.
+* SEQUENCE_TABLE(n) [AS] alias
 
-The result set is a single column with the predefined column name `value` of type `BIGINT UNSIGNED`. You can reference the `value` column in `SELECT` statements. The following statements are valid.
+
+```{.text .no-copy}
+SELECT … FROM SEQUENCE_TABLE(n) [AS] alias
+
+SEQUENCE_TABLE(n>) [AS] alias
```
Comment (percona-ysorokin) [inline docs/sequence-table.md]:
```suggestion
SEQUENCE_TABLE(n) [AS] alias
```

File: docs/sequence-table.md [hunk 9]
```diff
@@ -1,40 +1,70 @@
...
-The first number in the series, the initial term, is defined as `0` and the series ends with a value less than `n`. In this example, enter the following statement to generate a sequence:
+The first number in the series, the initial term, is defined as `0`, and the series ends with a value less than `n`.
+
+## Example usage
+
+Using `SEQUENCE_TABLE()`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT * FROM 
+  (SELECT value FROM SEQUENCE_TABLE(5)) AS sequence_data;
```
Comment (percona-ysorokin) [inline docs/sequence-table.md]:
```suggestion
mysql> SELECT * FROM SEQUENCE_TABLE(5) AS sequence_data;
```

File: docs/sequence-table.md [hunk 10]
```diff
@@ -1,40 +1,70 @@
...
+```
+
+Using `PERCONA_SEQUENCE_TABLE()`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT * FROM 
+  (SELECT value FROM PERCONA_SEQUENCE_TABLE(5)) AS sequence_data;
```
Comment (percona-ysorokin) [inline docs/sequence-table.md]:
```suggestion
mysql> SELECT * FROM PERCONA_SEQUENCE_TABLE(5) AS sequence_data;
```

**Review by percona-ysorokin — APPROVED** [review]
LGTM with 3 suggestions applied

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**